src/main/java/io/codelex/classesandobjects/practice/Account.java

```diff
@@ -21,6 +21,12 @@ public double balance() {
         return balance;
     }
 
+    public static void transfer(Account from, Account to, double howMuch) {
```

Comment:
Ir ļoti āķīgi šo metodi likt Account klasē. It kā var to pieņemt, bet šī ir balansēšana uz naža asmens vai tas ir/nav ok. Bet nu vari atstāt.